src/components/tests/GeoPushpinChart.spec.tsx

```diff
@@ -153,4 +152,45 @@ describe("Geo Pushpin Chart", () => {
         expect(wrapper.prop("afm")).toEqual(expectedAfm);
         expect(wrapper.prop("resultSpec")).toEqual(expectedResultSpec);
     });
+
+    it("should render component with Location and Size, Color has ignored ratio", () => {
+        const sizeRatio: VisualizationObject.IMeasure = Model.measure("measure_size")
+            .localIdentifier("size")
+            .ratio();
+        const colorRatio: VisualizationObject.IMeasure = Model.measure("measure_color")
+            .localIdentifier("color")
+            .ratio();
```

Comment:
Please reuse [this](https://github.com/gooddata/gooddata-react-components/pull/1392/files#diff-216e475df086a3b55df31b031d3b85fdR20)

Reply:
fixed